charts/scorpio-broker-dist/values.yaml

```diff
@@ -0,0 +1,982 @@
+## namespace for deployment
+namespace: scorpio-broker
```

Comment:
Still, setting the namespace that way needs to be avoided. namespaces are handled by helm.

Reply:
We have incorporated the review comment. Please review again